packages/cli/package.json

```diff
@@ -22,6 +22,7 @@
   "scripts": {
     "build": "node scripts/build.js",
     "build-release": "node scripts/build-release.js",
+    "dev": "node scripts/build.js --watch",
```

Comment:
I don't think having a watch mode for the CLI makes sense. You have to run the CLI via the command line, and it's pretty simple to add a `pnpm run build` in front of it before running it.

The cost on memory usage is not worth it IMO.

Reply:
True - I'll remove this.